packages/node_modules/webex/src/config.js

```diff
@@ -5,13 +5,16 @@
 import storageConfig from './config-storage';
 
 export default {
-  hydra: process.env.HYDRA_SERVICE_URL || 'https://api.ciscospark.com/v1',
```

Comment:
@taymoork2 Do you know why we are removing this?

Reply:
Add it back...?
Original config used `hdrya` and `hyrdaServiceURL`, while the meetings config used just `hydraServiceURL`

Reply:
It looks like we need to add it back until fully deprecating `phone`.